doc/classes/BaseMaterial3D.xml

```diff
@@ -501,22 +501,22 @@
 			Represents the size of the [enum TextureParam] enum.
 		</constant>
 		<constant name="TEXTURE_FILTER_NEAREST" value="0" enum="TextureFilter">
-			The texture filter reads from the nearest pixel only. The simplest and fastest method of filtering, but the texture will look pixelized.
+			The texture filter reads from the nearest pixel only. This makes the texture look pixelated from up close, and grainy from a distance (due to mipmaps not being sampled).
 		</constant>
 		<constant name="TEXTURE_FILTER_LINEAR" value="1" enum="TextureFilter">
-			The texture filter blends between the nearest 4 pixels. Use this when you want to avoid a pixelated style, but do not want mipmaps.
+			The texture filter blends between the nearest 4 pixels. This makes the texture look blurry from up close, and grainy from a distance (due to mipmaps not being sampled).
```

Comment:
Would it make sense to say ``smooth from up close`` instead of ``blurry``?

In my mind "blurry" really only makes sense for pixelated textures, for higher res textures its not really accurate.